odl/datasets/util.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Utilities for ODL datasets."""
+
+import os
+from os.path import join, expanduser, exists
+
+try:
+    # Python 2
+    from urllib2 import urlopen
+except ImportError:
+    # Python 3+
+    from urllib.request import urlopen
+    
+from shutil import copyfileobj
+from scipy import io
+
+
+__all__ = ('get_data_dir', 'get_data')
+
+
+def get_data_dir():
+    """Get the data directory."""
+    data_home = expanduser(join('~', 'odl_data'))
```

Comment:
I'd find `ODL_DIR` confusing since it could also mean that ODL is installed there. Not sure what would be a better name, suggestions:
- `ODL_CONFIG_DIR`
- `ODL_DATA_DIR`
- `ODL_PREFIX`
- `ODL_HOME`
- `ODL_LOCAL_DIR`

Can you also add a dot before `odl`? It's not unlikely that people have the ODL repository in `~/odl` so we shouldn't pollute that by default.